Models/Dbos/AtualizacaoDbo.cs

```diff
@@ -4,6 +4,7 @@ public class AtualizacaoDbo
     {
         public int id_atualizacao { get; set; }
         public int id_consultor { get; set; }
+        public int ?id_investimento { get; set; }
```

Comment:
Literalmente só um errinho com o nullable aqui
```suggestion
        public int? id_investimento { get; set; }
```